lib/apivore/validator.rb

```diff
@@ -20,9 +21,11 @@ def matches?(swagger_checker)
       unless has_errors?
         send(
           method,
-          full_path(swagger_checker),
-          params['_data'] || {},
-          params['_headers'] || {}
+          *RailsShim.action_dispatch_request_args(
+            full_path(swagger_checker),
+            params: params[:_data] || {},
```

Comment:
We should keep these as string keys for now. Don't think we can rely on the input to be a hash with indifferent access.